media-gfx/blender/blender-2.83.15.ebuild

```diff
@@ -302,8 +352,9 @@ pkg_postinst() {
 	ewarn "  https://developer.blender.org/"
 	ewarn
 
-	if use python_single_target_python3_8; then
-		elog "You've enabled python-3.8 support for blender, which is still experimental."
+	if use !python_single_target_python3_7; then
```

Comment:
```suggestion
	if ! use python_single_target_python3_7; then
```

Reply:
3_7 isn't a valid target anymore (i.e. even if you have it in COMPAT, it won't show up or be usable), so any references to 3_7 should be removed instead.